packages/shared-ux/chrome/serverless_project_navigation.mdx

```diff
@@ -19,13 +19,10 @@ Welcome to the serverless project navigation documentation. Our tools help teams
         - [`NavigationTreeDefinition`](#navigationtreedefinition)
         - [`GroupDefinition`](#groupdefinition)
         - [`RecentlyAccessedDefinition`](#recentlyaccesseddefinition)
-    - [React components](#react-components)
-      - [`unstyled`](#unstyled)
     - [Important concepts](#important-concepts)
       - [Deep links](#deep-links)
       - [Cloud link](#cloud-links)
       - [Preconfigured navigation sections](#preconfigured-navigation-sections)
-      - [Active navigation path](#active-navigation-path)
```

Comment:
"Active navigation path" is still in the docs, we should keep it in the toc

Reply:
Ups, great catch. Addressed in https://github.com/elastic/kibana/pull/174544/commits/45de12417456fd37877cde6304c5741968bccb49